specs/GetEnvironment.md

```diff
@@ -0,0 +1,105 @@
+# Background
+When the WebView2 team was making design changes to the [WebResourceRequested API](https://github.com/MicrosoftEdge/WebView2Feedback/wiki/WebResourceRequested-API-Review-Spec) for .NET, we realized it was not intutive to create the WebView2 Environment. Thus it was not straightforward to use the CreateWebResourceResponse off the WebView2 Environment.
+WebView2 team plans to introduce an easier way to get the Environment variable and CreateWebResourceResponse.
+
+# Description
+One can get WebView2 Environment from `Environment`. 
```

Comment:
`...get the CoreWebView2Environment used to create the CoreWebView2 from that CoreWebView2's CoreWebView2Environment property.`